src/video_analyze.py

```diff
@@ -181,6 +185,8 @@ def video_analyze(infile, width, height, ref_fps, pixel_format, luma_threshold,
             continue
         if debug > 2:
             print(f"video_analyze: read image value: {value_read}")
+            if value_read is None:
+                cv2.imwrite(f"debug/{infile}_{frame_num}.png", img)
```

Comment:
do you want to hardcode debug/ here? What happens if debug/ does not exist?

I'd suggest an option for passing this directory, or use a temp directory otherwise:
```
tempfile.mkdtemp(prefix='metiq.debug..', dir='/tmp')
```

Reply:
Nothing will be written.